bin/generate-readme.js

```diff
@@ -124,14 +124,13 @@ async function generateReadme (repoMetadata) {
 }
 
 async function getRepos () {
-  const q = 'nodejs in:.repo-metadata.json org:googleapis is:public archived:false';
-  let url = new URL('/search/repositories', baseUrl);
-  url.searchParams.set('q', q);
+  let url = new URL('/orgs/googleapis/repos', baseUrl);
+  url.searchParams.set('type', 'public');
   url.searchParams.set('per_page', 100);
   const repos = [];
   while (url) {
     const res = await github.request({ url: url.href });
-    repos.push(...res.data.items.map(r => r.full_name));
+    repos.push(...res.data.filter(r => (r.language === 'TypeScript' || r.language === 'JavaScript') && r.archived === false).map(r => r.full_name));
```

Comment:
I think we want to limit this to `r.private === false` as well.  I'm a little pleasantly surprised `r.language` was right, given the acrobatics I had to go through in the repo-automation-tools repository.